packages/@o3r-training/tools/cli/extract-folder-structure/extract-folder-structure.ts

```diff
@@ -0,0 +1,45 @@
+import {lstat, readdir, readFile, writeFile} from 'node:fs/promises';
+import {type FileSystem, getFilesTree} from '@o3r-training/tools';
+import {join, resolve} from 'node:path';
+import {program} from 'commander';
+
+program
+  .description('Extract folder structure')
+  .requiredOption('--files <files>', 'List of files and folder to extract in addition to the path')
+  .option('-r, --root <root>', 'Root of the extraction')
+  .option('-o, --output <output>', 'Output folder path')
+  .parse(process.argv);
+
+const options: any = program.opts();
+const cwd = options.root ? resolve(process.cwd(), options.root) : process.cwd();
```

Comment:
is it working if `root` is an absolute path ?

Reply:
Yes I tested with the showcase SDK by setting `root` to the absolute path of @ama-sdk/showcase-sdk and the correct file was generated in the showcase-sdk folder